components/map/Clusters.js

```diff
@@ -32,15 +33,15 @@ export default function Clusters({users}) {
   })
 
   const { clusters, supercluster } = useSupercluster({
-    points: users,
+    points: [...users,...events],
```

Comment:
This needs to be done in `getStaticProps` to ensure it is done on the server & not the client.

Reply:
Here is the filtering code:
```js
const filterData = (value) => {
    const valueLower = value.toLowerCase();
    const terms = [...updateSelectedTagsFilter(value)];

    results = users.filter((user) => {
      if (user.properties.name.toLowerCase().includes(valueLower)) {
        return true;
      }

      let userTags = user.properties.tags?.map((tag) => tag.toLowerCase());

      if (terms.every((keyword) => userTags?.includes(keyword.toLowerCase()))) {
        return true;
      }

      return false;
    });

    setFilteredUsers(results);
  };
```

Assuming that users & events are merged into the users array, the filtering code will still work fine. Events have a name property & the code uses optional chaining to handle tags not existing.

The combined array should be renamed though.

Reply:
Done in new commit